lib/rubocop/token.rb

```diff
@@ -14,7 +14,8 @@ def self.from_parser_token(parser_token)
     def initialize(pos, type, text)
       @pos = pos
       @type = type
-      @text = text
+      # Parser token "text" may be a Fixnum
+      @text = text.is_a?(String) ? text : text.to_s
```

Comment:
Why? In a blind pursuit of speed and performance, I wanted to avoid string copies. But does `String#to_s` actually make a copy?

```c
static VALUE
rb_str_to_s(VALUE str)
{
    if (rb_obj_class(str) != rb_cString) {
	return str_duplicate(rb_cString, str);
    }
    return str;
}
```

Doesn't look like it. Too bad for me.